app/src/main/java/com/sunasterisk/weather/screen/MainActivity.kt

```diff
@@ -2,12 +2,27 @@ package com.sunasterisk.weather.screen
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.fragment.app.FragmentManager
 import com.sunasterisk.weather.R
+import com.sunasterisk.weather.data.model.element.Screen
+
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var fragmentManager: FragmentManager
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        initView()
+        initData()
+    }
+
+    private fun initView() {
+        fragmentManager = supportFragmentManager
+        fragmentManager.beginTransaction()
+                       .add(R.id.container, Screen.WEATHER.fragment, Screen.WEATHER.name)
```

Comment:
```suggestion
                       .add(R.id.container, WeatherFragment.newInstance(), WeatherFragment.tag)
```
https://github.com/framgia/coding-standards/blob/master/vn/android/codingstyleguide.md
xem mục **2.11**